api/src/endpoints/postJoin.js

```diff
@@ -0,0 +1,70 @@
+const mongoose = require("mongoose");
+const { nodesSchema, usersSchema } = require("../mongooseSchemas");
+const nodes = mongoose.model("nodes", nodesSchema);
+const users = mongoose.model("users", usersSchema);
+
+const postJoin = {
+  method: "POST",
+  url: "/api/join/:userId",
+  schema: {
+    body: {
+      type: "object",
+      properties: {
+        id: { type: "string" },
```

Comment:
change to number

Reply:
Actually we can't. The type it's ObjectId.